javascript-functions/main.js

```diff
@@ -0,0 +1,30 @@
+// function 1
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  return console.log('convertMinutesToSeconds output:', seconds);
```

Comment:
Approved for this one, but make sure to stick to clean return values in the future!